src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,33 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({
+                'is-active': tab === (selectedTab || tabs[0]),
+              })}
+              data-cy="Tab"
+              key={tab.id}
+              onClick={() => tab !== selectedTab && onTabSelected(tab)}
```

Comment:
```suggestion
              onClick={() => onTabSelected(tab.id)}
```

Reply:
Here test also wouldn't work, because I have to pass an object to a function..
And also I have to save the condition in order to not invoke onTabSelected function if the tab is selected already (that's what tests ask for)